src/pages/about/_leaders.js

```diff
@@ -227,7 +227,7 @@ const DescriptionWrapper = styled.div`
 `
 
 const LeaderWrapper = styled(Flex)`
-    cursor: pointer;
+    cursor: default;
```

Comment:
you can remove cursor properties at once

Reply:
if  remove the cursor property it will have a diff pointer something like capslock I if hover to text (regular behavior),
should I keep it this way (cursor default), or just remove it ?